AcadeMeData/test_msgtags.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+from AcadeMeData.models import MessageTags, Messages, User
+
+
+@pytest.fixture
+def user_example():
+    user_data = {'username': "username22123", 'password': "password", 'email': "user@example.com",
+                 'university': "RU",
+                 'degree': "CS"}
+    user = User.create_user(*user_data)
+    return user
+
+
+@pytest.fixture
+def generate_message(user_example, msgID=10, text='This is a test message yo'):
+    user = user_example
```

Comment:
no need 
search for user_example. the fixture return user, 
you can call it like that user_example it is holding it for you as User

Reply:
You're right 
Havent noticed in this file
FIXED